p/scripts/main.js

```diff
@@ -785,6 +785,7 @@ function auto_share(key) {
 let box_to_follow;
 
 function onScroll() {
+	on_scroll_header();
```

Comment:
That sucks terribly imho because the point of this function is not to batter the server with requests.

Reply:
No, the point was to avoid very fast firing events taking a lot of CPU

Reply:
Fair enough. (My poor server suffers more from the lot of CPU thing.) Scroll events already seem to be limited to some 60 fps despite the polling rate being higher afaik. Personally I'd much rather not have it than have it laggy.

If you're worried that scroll events might exceed the framerate you could do something like this with `requestAnimationFrame`, maybe coupled with a limitation to e.g. 30 fps. That is, crucially, closer to 50 ms than to 500 ms.

```js
let ticking = false;

function onScrollRequestAnimationFrame() {
  if (!ticking) {
    window.requestAnimationFrame(() => {
      // logic
      ticking = false;
    });
    ticking = true;
  }
}
```